lib/manager/pip-compile/artifacts.ts

```diff
@@ -0,0 +1,102 @@
+import { quote as pipCompile } from 'shlex';
+import { TEMPORARY_ERROR } from '../../constants/error-messages';
+import { logger } from '../../logger';
+import { ExecOptions, exec } from '../../util/exec';
+import { deleteLocalFile, readLocalFile, writeLocalFile } from '../../util/fs';
+import { getRepoStatus } from '../../util/git';
+import type {
+  UpdateArtifact,
+  UpdateArtifactsConfig,
+  UpdateArtifactsResult,
+} from '../types';
+
+function getPythonConstraint(
+  config: UpdateArtifactsConfig
+): string | undefined | null {
+  const { constraints = {} } = config;
+  const { python } = constraints;
+
+  if (python) {
+    logger.debug('Using python constraint from config');
```

Comment:
```suggestion
    logger.debug('Using Python constraint from config');
```

Do we capitalize proper nouns like Python, JavaScript in our debug statements?

Reply:
The constraint is case sensitive and lower case, so I prefer to keep it as-is